Malayalam/malayalam.md

```diff
@@ -1 +1,3 @@
-ഞാൻ ഒരു മലയാളി. ഒരു സാഹസിക യാത്രയുടെ തുടക്കം.
\ No newline at end of file
+ഞാൻ ഒരു മലയാളി. ഒരു സാഹസിക യാത്രയുടെ തുടക്കം.
+
+ഇത് ഒരു നല്ല [കുട്ടിയുടെ](Malayalam/boy/life.md) കഥയാണ്
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.